src/main/kotlin/org/rust/ide/console/RsConsoleProcessHandler.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.console
+
+import com.intellij.execution.process.KillableColoredProcessHandler
+import com.intellij.openapi.Disposable
+import com.intellij.openapi.util.Disposer
+import com.intellij.openapi.util.Key
+import com.intellij.util.io.BaseOutputReader
+import java.nio.charset.Charset
+
+class RsConsoleProcessHandler(
+    process: Process,
+    private val myConsoleView: RsConsoleView,
```

Comment:
`my` prefix is not necessary, you don't have to use it in Kotlin code (in other files too)